controllers/mealsController.js

```diff
@@ -55,7 +65,10 @@ const index = async (req, res) => {
     const meals = await findUserMeals(user.id);
     const mealsResults = await findAllMealFoods(meals, user.id)
     res.setHeader("Content-Type", "application/json");
-    res.status(201).send(JSON.stringify({ user_id: `${user.id}`, meals: mealsResults }) )
+    res.status(201).send(JSON.stringify({
+      user_id: `${user.id}`,
+      meals: mealsResults
+    }));
   } catch (error) {
     res.setHeader("Content-Type", "application/json");
     res.status(400).send(JSON.stringify("Could not fetch meals."))
```

Comment:
These are good changes--thanks for cleaning these up. Should we add the same formatting to this error?
`"error": "Could not fetch meals."`

Reply:
Good idea, I'll do that now.